tests/cache_test.py

```diff
@@ -0,0 +1,62 @@
+
+import random
+from time import sleep
+
+from superdesk.cache import cache
+from superdesk.tests import TestCase
+from bson import ObjectId
+
+
+class Foo():
+
+    def __init__(self):
+        self.test_calls = 0
+
+    @cache(ttl=3, tags=('count',))
+    def count_calls(self):
+        self.test_calls += 1
+        return self.test_calls
+
+    @cache(ttl=5, tags=('random',))
+    def random(self):
+        return random.random()
+
+    @cache(ttl=5)
+    def identity(self, identity):
+        return identity
+
+
+foo = Foo()
+
+
+class CacheTestCase(TestCase):
+
+    def test_cache(self):
+        self.assertEqual(1, foo.count_calls())
+        self.assertEqual(1, foo.count_calls(), 'not using cache')
+        sleep(4)
```

Comment:
Please, use here smaller time 1-2sec, because of synchronous running this test case adds ~5sec and it is too much for one test case.